### PR TITLE
add get interaction with api

### PR DESCRIPTION
**Pull request by lbhMichael**

### CODE REVIEW

File: src/components/apiInformationLink/apiInformationLink.component.jsx [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
-        return(
-            <a className="lbh-link lbh-link--no-visited-state" href={url}>
-                {linkText}
-            </a>
-        );
-    };
-    
+const ApiInformationLink = ({ linkText, url }) => {
+  if (!url) {
+    return <p>{linkText} (Link To Be Confirmed)</p>;
```
Comment (humulla) [inline src/components/apiInformationLink/apiInformationLink.component.jsx:5]: Why do we have 'Link to be confirmed' here?

**Comment by humulla** [comment]
Also, please resolve the conflicts and the issues with the pipeline.

File: cypress/integration/viewApiInformation.spec.js [hunk 2]
```diff
@@ -1,193 +1,268 @@
...
     });
 
-    it("Can choose API version", function() {
-        const selectedVersion = "2.0.0";
-        cy.get('select').select(selectedVersion);
-        cy.get('select#VersionNo option:selected').should('have.text', selectedVersion);
-    });
+    // TODO: revise test according to what is expected. The TestAPI has 3 applications
+    // TODO: add further tests taking into consideration authorisation for editing applications
+    // it(`View applications consumed by an API on ${screenSize} screen`, function() {
```
Comment (humulla) [inline cypress/integration/viewApiInformation.spec.js:108]: Why is this test commented out? 
We should have a test to view applications consumed by an API

Comment (lbhMichael) [reply cypress/integration/viewApiInformation.spec.js:108]: I found the test commented out when I started writing the tests then I played with it but then left it commented out. Its not needed

Comment (humulla) [reply cypress/integration/viewApiInformation.spec.js:108]: I personally think this test is now needed as we need to ensure that a user is able to successfully view the applications consumed by section. Please add this test in

**Review by humulla — APPROVED** [review]

**Pull request merged**